app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionFeedbackFragment.java

```diff
@@ -100,11 +106,15 @@ public void onClickSubmitFeedback() {
 
     public void onSubmitSuccess() {
         // TODO: show success action
+        if (isDetached())
+            return;
         Toast.makeText(getContext(), "submit success", Toast.LENGTH_SHORT).show();
     }
 
     public void onSubmitFailure() {
         // TODO: show failure action
+        if (isDetached())
```

Comment:
You called `compositeDisposable.clear()` in `onDeach()`.
Is this `isDetached()` necessary here?